packages/conform-validator/test/valibot.test.ts

```diff
@@ -0,0 +1,111 @@
+import type { ExtractSchema, ParsedFormValue } from 'hono/types'
+import type { Equal, Expect } from 'hono/utils/types'
+import type { StatusCode } from 'hono/utils/http-status'
+import * as v from 'valibot'
+import { Hono } from 'hono'
+import { hc } from 'hono/client'
+import { parseWithValibot } from 'conform-to-valibot'
+import { conformValidator } from '../src'
+
+describe('Validate requests using a Valibot schema', () => {
+  const app = new Hono()
+
+  const schema = v.object({
+    name: v.string(),
+    age: v.pipe(
+      v.string(),
+      v.transform((v) => Number(v)),
+      v.integer()
+    ),
+    nickname: v.optional(v.string()),
+  })
+
+  const route = app.post(
+    '/author',
+    conformValidator((formData) => parseWithValibot(formData, { schema })),
+    (c) => {
+      const submission = c.req.valid('form')
+      const value = submission.value
+
+      return c.json({
+        success: true,
+        message: `${value.name} is ${value.age}, nickname is ${
+          value?.nickname || 'nothing yet :3'
+        }`,
+      })
+    }
+  )
+
+  it.skip('check the route object types', () => {
```

Comment:
Is this `skip` intended?

Reply:
No, so I removed it. Thank you for pointing that out.